crates/sui-core/src/traffic_controller.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2021, Facebook, Inc. and its affiliates
```

Comment:
was this code really from Facebook or is this new and it shouldn't have this header?